From c4828edd9dae2d606a693788d7724b8411722f19 Mon Sep 17 00:00:00 2001
From: Heming Zhao <heming.zhao@suse.com>
Date: Fri, 8 Oct 2021 09:33:35 +0800
Subject: [PATCH 07/14] dlm_tool man: add command "joinleave" & "USAGE"

add missing command "joinleave".
add "USAGE" section to show detail usage.

Signed-off-by: Heming Zhao <heming.zhao@suse.com>
---
 dlm_tool/dlm_tool.8 | 80 +++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 80 insertions(+)

diff --git a/dlm_tool/dlm_tool.8 b/dlm_tool/dlm_tool.8
index bbc11c261b7e..7e82f8908ed6 100644
--- a/dlm_tool/dlm_tool.8
+++ b/dlm_tool/dlm_tool.8
@@ -48,6 +48,10 @@ dlm_tool \- a utility for the dlm and dlm_controld daemon
 .br
 	Leave a lockspace.
 
+.BI joinleave " name"
+.br
+	Do two actions: first Join then Leave a lockspace. (mainly for test)
+
 .BI lockdebug " name"
 .br
 	Complete display of locks from the lockspace.
@@ -112,6 +116,82 @@ Print help, then exit
 .B \-V
 Print program version information, then exit
 
+.SH USAGE
+
+.B dlm_tool ls [-n] [ls_name]
+
+-
+
+.B dlm_tool status [-v]
+
+-
+
+.B dlm_tool dump
+
+-
+
+.B dlm_tool dump_config
+
+-
+
+.B dlm_tool fence_ack \fInodeid\fP
+
+* run "dlm_tool status -v" to get \fInodeid\fP
+
+-
+
+.B dlm_tool log_plock
+
+-
+
+.B dlm_tool plocks \fIls-name\fP
+
+-
+
+.B dlm_tool join [-e 0|1] [-f 0|1] \fIls-name\fP
+
+-
+
+.B dlm_tool leave \fIls-name\fP
+
+-
+
+.B dlm_tool joinleave [-e 0|1] [-f 0|1] \fIls-name\fP
+
+-
+
+.B dlm_tool lockdebug [-s] [-v] [-w] \fIls-name\fP
+
+-
+
+.B dlm_tool lockdump [-M] \fIls-name\fP
+
+-
+
+.B dlm_tool run|run_start [-n] \fIcommand\fP
+
+* current run & run_start support \fIcommand\fP:
+.br
+  - "lvm lvchange --refresh"
+.br
+  - "lvm lvs"
+
+-
+
+.B dlm_tool run_check|run_cancel [-i "sec"] \fIuuid\fP
+
+-
+
+.B dlm_tool run_list
+
+-
+
+.B dlm_tool -h
+
+-
+
+.B dlm_tool -V
+
 .SH SEE ALSO
 .BR dlm_controld (8),
 .BR dlm.conf (5)
-- 
2.33.0

